kebechet/managers/pipfile_requirements/pipfile_requirements.py

```diff
@@ -82,6 +82,19 @@ def get_pipfile_lock_requirements(self, content_str: str) -> typing.Set[str]:
 
         return requirements
 
+    def _create_missing_pipenv_files_issue(self, file_name):
+        issue_title = (
+            f"Kebechet Pipfile Requirements Manager: no {file_name} found in repo"
+        )
+        body = f"""Kebechet pipfile_requirements manager is installed but no {file_name} was found.
+
+        Either remove this manager from .thoth.yaml or use pipenv for requirements management.
```

Comment:
I feel that the message could be expanded a bit more, to help (newer) users to better understand the problem and its resolution.

Maybe something along these lines:

```suggestion
        body = f"""Kebechet pipfile_requirements manager is installed but no `{file_name}` was found in this repository.
        
        `{file_name}` is required by the pipfile_requirements manager in its current configuration (as specified in `.thoth.yaml`).

        Either remove this manager from `.thoth.yaml`, adjust its configuration, or update the repository to meet the requirements.
        
        Reference: see the documentation for [pipfile_requirements](https://thoth-station.ninja/docs/developers/kebechet/managers/pipfile_requirements.html).
```

but feel free to improve